samples/highcharts/chartchooser/continuous-distribution-scatter/demo.html

```diff
@@ -0,0 +1,8 @@
+
+<script src="https://code.jquery.com/jquery-3.1.1.min.js"></script>
```

Comment:
Remove jQuery. There's also one other sample where it is referred and not used.

Reply:
You are right. The other demo is continuous-distribution-density-chart (corrected as well)